pkgs/development/python-modules/qcelemental/default.nix

```diff
@@ -6,27 +6,30 @@
 , numpy
 , pint
 , pydantic
+, poetry-core
 , pytestCheckHook
 , pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "qcelemental";
-  version = "0.25.1";
-  format = "setuptools";
+  version = "0.26.0";
+
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-4+DlP+BH0UdWcYRBBApdc3E18L2zPvsdY6GTW5WCGnQ=";
+    hash = "sha256-oU6FEM2/2mRe8UYcGv0C77WZMRcz27pfg/zR1haKbd0=";
   };
 
   propagatedBuildInputs = [
     networkx
     numpy
     pint
     pydantic
+    poetry-core
```

Comment:
Could you please move `poetry-core` to `nativeBuildInputs`?
It's a build-time dependency but not a runtime dependency.